$MirOS: ports/devel/sdl/patches/patch-src_video_x11_SDL_x11gl_c,v 1.2 2005/10/02 12:01:54 tg Exp $
$OpenBSD: patch-src_video_x11_SDL_x11gl_c,v 1.6 2003/09/29 05:41:37 pvalchev Exp $
--- src/video/x11/SDL_x11gl.c.orig	Mon May 16 08:23:12 2005
+++ src/video/x11/SDL_x11gl.c	Fri Nov  3 19:52:23 2006
@@ -34,8 +34,9 @@ static char rcsid =
 #include "SDL_x11dga_c.h"
 #include "SDL_x11gl_c.h"
 
-#if defined(sgi)
+#if defined(sgi) || defined(__OpenBSD__)
 /* IRIX doesn't have a GL library versioning system */
+/* OpenBSD and MirOS BSD dlopen(3) says we don't need a major */
 #define DEFAULT_OPENGL	"libGL.so"
 #else
 #define DEFAULT_OPENGL	"libGL.so.1"
